source/administration/incident-response-application.rst

```diff
@@ -0,0 +1,667 @@
+
+    "name", "The name of the step", "string", "Yes. This field must be unique."
+    "start_actions", "Actions to perform when the step is reached.", "[]Action", "If ``finished_actions`` not set, yes"
+    "finish_actions", "Actions to perform when the step is finished.", "[]Action", "If ``start_actions`` not set, yes"
+
+Note that the step name must be a sequence of one or more letters, digits and underscores, and start with a letter.
+
+Action
+^^^^^^^^^^^^^^^
+
+Actions are performed when steps are started and finished.
+
+.. csv-table::
+    :header: "Field", "Description", "Type", "Required"
+
+    "name", "The name of the action", "string", "Yes. This field must be unique."
+    "type", "The type of action to perform", "string", "Yes"
+    "*additional fields", "Additional fields based on the type of action. See below for more details.", "string", "Yes"
+
+Note that the action name must be a sequence of one or more letters, digits and underscores, and start with a letter.
+
+Create Channel (type: ``create_channel``)
+*******************************************
+
+Creates a channel with the given parameters.
+
+.. csv-table::
+    :header: "Field", "Description", "Type", "Required"
+
+    "channel_name", "The name of the channel. This is the channel handle used in the URL.", "string", "Yes"
+    "channel_displayname", "The display name of the channel.", "string", "Yes"
+    "team_name", "The team to create the channel in. If none specified, the channel is created in the same team as where the app workflow was triggered.", "string", "No"
+    "header", "The channel header.", "string", "No"
+    "purpose", "The channel purpose.", "string", "No"
+    "private", "``Yes`` if the channel is private, ``No`` if the channel is public.", "boolean", "No"
+
+Below is an example JSON for a step containing one ``create_channel`` start action.
+
+.. code-block:: json
+
+  "steps": [
+      {
+          "name": "Setup",
+          "start_actions": [
+              {
+                  "name": "CreateWarroom",
+                  "type": "create_channel",
+                  "channel_name": "system-incident-{{.Instance.Number}}",
+                  "channel_displayname": "System Incident {{.Instance.Number}}",
+                  "header": "[Nagios Monitoring Dashboard](http://18.188.56.242/nagios/)"
+              },
+          ]
+      }
+  ]
+
+.. tip::
+  If the ``create_channel`` action attempts to create a channel that already exists, the workflow fails to continue as it's unable to create that said channel.
+  
+  Therefore, it is highly recommended that you use instance template variables to define ``channel_name`` and ``channel_displayname`` parameters to avoid non-unique channel names. For instance, if you define the channel name to be ``system-incident-{{.Instance.Number}}`` such as in the app workflow JSON example above, channels are created with names ``system-incident-1``, ``system-incident-2``, and so forth, avoiding duplicate names.
```

Comment:
Recommending additional simplification of this content (hoping I didnt lose too much meaning in my changes):  

It is highly recommended that you use instance template variables to define ``channel_name`` and ``channel_displayname``.  For instance, defining the channel name in your workflow as``system-incident-{{.Instance.Number}}`` will create channels with names ``system-incident-1``, ``system-incident-2``each time the workflow runs, thereby ensuring unique names and preventing duplicate names.